src/proc/zero-order.cpp

```diff
@@ -397,7 +400,6 @@ namespace librealsense
         {
```

Comment:
auto data = f.as<rs2::frameset>(); is potentially a nullptr.
Pls wrap it into conditional clause